src/views/misc/Terms.vue

```diff
@@ -0,0 +1,127 @@
+<p>In the event of any termination of your access to MorphoBank or the Services by Phoenix, all licenses granted to you will terminate, including, without limitation, your license to access and use MorphoBank, the Services and the Materials. Notwithstanding such termination, all other terms and conditions of this Terms of Use will remain in force.</p>
+    <h2>7. Privacy</h2>
+    <p>Phoenix will use reasonable efforts to protect the privacy of users and any personally identifiable information collected from users. By utilizing MorphoBank and the Services, you consent to being named as a scientific author and contributor. For planning, scientific review, and marketing purposes, Phoenix will keep records of usage for the purpose of compiling aggregate usage information. Logs of usage may also be maintained for the purposes of monitoring and improving services or measuring the impact of a particular function on our resources. You hereby consent to the use of a Google Analytics cookie or similar monitoring tool which collects usage data regarding your use of the Service. You will be required to supply your name, the name of the institution with which you are affiliated, and a unique user name and password along with a valid email address for account registration. Students must register their advisor’s name. In the event that students who wish to register have no advisor, please use our <router-link to="/askus">Contact Us</router-link> form to request approval. All other personal information is supplied by you on a voluntary basis.</p>
+    <p>MorphoBank is currently offered free of charge and Phoenix does not store or record any credit card, debit card, bank account, or any other form of payment account information on its servers. If Phoenix elects to charge for access to MorphoBank or the Services in the future, these Terms of Use will be revised to reflect any such new payment methodology.</p>
+    <p>Phoenix may share with third parties without restriction aggregate information or individual information that does not personally identify a user, including without limitation Google Analytics. You are also hereby notified that Phoenix may be required to disclose individually identifiable personal information about users of MorphoBank or the Services (i) if required by court order or applicable law; (ii) if necessary to perform or enforce these Terms of Use; (iii) in order to detect, prevent, or otherwise address fraud, security or technical issues; (iv) in order to respond to your support requests and improve services; (v) in order to protect the rights, property or safety of Phoenix, its users and the public; or (vi) as necessary to enable third parties such as contractors to facilitate assistance in maintaining, improving or operating MorphoBank or the Services, provided all such third parties agree in writing to keep such information confidential.</p>
```

Comment:
Can you split the conditions into multiple lines (i,ii,iii,iii) so it is consistent with the format above?